x-pack/elastic-agent/elastic-agent.reference.yml

```diff
@@ -135,3 +135,7 @@ datasources:
 # Available log levels are: error, warning, info, debug
 #logging.level: trace
 
+# Sets output of logging. Available options are `console`, `file`.
+# File output is located at `data/logs/elastic-agent`
+# logging.output: console
+
```

Comment:
The reference file should be more complete https://github.com/elastic/beats/blob/master/libbeat/_meta/config/logging.reference.yml.tmpl

Reply:
what i dont like in `to_xxx` configuration options is that it evokes that you can set true to more than one and it will work, having a single option with a single value makes it clear there's no way of having 2 outputs of logging. 
i will introduce some more options or i can even make it 1;1 and let libbeat logger handle that without any verification from our side.

Reply:
I do not like it either but I am into moving responsibility out if possible, @urso is there a plan to change the logging options?